pkg/graph/schema.graphql

```diff
@@ -264,6 +264,144 @@ type CreateAccessibilityRequestDocumentPayload {
   userErrors: [UserError!]
 }
 
+"""
+The shape of a solution for a business case
+"""
+type BusinessCaseSolution {
+  acquisitionApproach: String
+  cons: String
+  costSavings: String
+  hasUi: String
+  hostingCloudServiceType: String
+  hostingLocation: String
+  hostingType: String
+  pros: String
+  securityIsApproved: Boolean
+  securityIsBeingReviewed: String
+  summary: String
+  title: String
+}
+
+"""
+enum 
+"""
+enum LifecycleCostPhase {
+  """
+  development
+  """
+  DEVELOPMENT
+  """
+  Operations and Maintentance
+  """
+  OPERATIONS_AND_MAINTENANCE
+  """
+  Other
+  """
+  OTHER
+}
+
+"""
+enum 
+"""
+enum LifecycleCostSolution {
+  """
+  A
+  """
+  A
+  """
+  As is
+  """
+  AS_IS
+  """
+  B
+  """
+  B
+  """
+  Preferred
+  """
+  PREFERRED
+}
+
+"""
+enum 
+"""
+enum LifecycleCostYear {
+  """
+  1
+  """
+  LIFECYCLE_COST_YEAR_1
+  """
+  2
+  """
+  LIFECYCLE_COST_YEAR_2
+  """
+  3
+  """
+  LIFECYCLE_COST_YEAR_3
+  """
+  4
+  """
+  LIFECYCLE_COST_YEAR_4
+  """
+  5
+  """
+  LIFECYCLE_COST_YEAR_5
+}
+
+"""
+The shape of an estimated lifecycle cost row
+"""
+type EstimatedLifecycleCost {
+  businessCaseId: UUID!
+  cost: Int
+  id: UUID!
+  phase: LifecycleCostPhase
```

Comment:
The enums for phase, solution, and year don't match with our constants in Go because they can't. The literal values for the years are 1, 2, 3, etc. So I don't think these types are actually being used. Maybe we can remove them for now?

Reply:
I'm curious about this... shouldn't they be able to match? Can the enum values be "1", "2" etc? Or does Graphql not allow that? Anyway, good catch, something worth keeping an eye on. We can remove them for now and put them back in when we actually need this model

Reply:
Ok. I took this code out and suddenly all kinds of tests are failing. I guess we can leave it in for now?